inference-engine/tests/functional/plugin/plaidml/shared_tests_instances/single_layer_tests/shuffle_channels.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "single_layer_tests/shuffle_channels.hpp"
+using LayerTestsDefinitions::ShuffleChannelsLayerTest;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+    InferenceEngine::Precision::I32,
+    InferenceEngine::Precision::FP32,
+};
+
+const std::vector<std::vector<size_t>> inputShapes = {
+    {5, 12, 200, 400},
+    {1, 30, 40, 40}
```

Comment:
I think we could make these inputs smaller to speed up the test and it would not hurt the coverage.

Reply:
ok